frontend/participant/templates/participant/appointments/appointments.html

```diff
@@ -71,7 +71,7 @@ <h5 class="card-title"> {{ appointment.experiment.name }}</h5>
             </div>
         </div>
         {% empty %}
-            {% translate 'appointments:content:no_appointments' %}
+            {% trans 'appointments:content:no_appointments' %}
```

Comment:
FIY `trans` is the old name of this tag and `translate` is a newer option. Doesn't matter much but I prefer to use `translate` in new code